### ISSUE TITLE
Dual-axis plotting fails

### ISSUE THREAD

**Issue by bilderbuchi** [nschloe/matplotlib2tikz#14]
If you make a dual-axis plot (analogous to matlab plotyy), the two separate axes are collapsed onto one another in the final output. 

Code:
```python
from matplotlib.pylab import *
import matplotlib2tikz as tt
fig = plt.figure()
ax = fig.add_subplot(111)

dat = linspace(0,10,20)
ax.plot(dat,dat**2)
ax2 = ax.twinx()
ax2.plot(20*dat,20*dat**2)
tt.save('fails.tikz')
```

**Comment by nschloe** [comment]
Fixed now.

**Comment by pylipp** [comment]
I had trouble using dual-axis plotting in combination with the ```ggplot``` style by MPL. 
This style comes with a gray, opaque background that will cover the first plot. Hence in the tex-output I moved the line ```axis background ...``` from the second to the first axis. 
Also the xticklabels of the plots are not aligned correctly in the output, so I set ```xtickslabels={,,}``` in one of the axis to hide it. 
Not requesting a fix, rather sharing my experience for others who might stumble across this.
